Update node version limt in the @gitbeaker/node release to match the limit outlined by Got (>=10.19.0)

Hi,

I am trying to install this project in my nodejs version `10.16.0`. It is failing due to ```got@11.3.0```.

Error logs of ```yarn add @gitbeaker/node```:
```bash
error got@11.3.0: The engine "node" is incompatible with this module. Expected version ">=10.19.0". Got "10.16.0"
error Found incompatible module.
```

Can't we use this library in `nodejs 10.16.0`? 

Actually I am in my organization laptop where I don't have admin permissions, so I am unable to update NodeJS here.

Please look into this, I am not giving this to Issue/Bug/Feature request, coz I am not sure what is the correct label for it.

Hi @jdalrymple,

I have already mentioned why I can't update. So I tried `gitbeaker/browser` before this in my `electron` application, it was working fine for me till today. Now I have started reviewing some security aspects of `electron`.

When I was using ```nodeIntegration: true``` in electron, it was working fine. Now I have changed this flag to false, and added preload script, for fixing the security issue as mentioned by electron team.

After doing this, I am not able to run anything regarding `gitbeaker/browser`.

```bash
Module not found: Error: Can't resolve '@gitbeaker/browser' in 'D:\MyApp\electron-react-ts\app\gitlab'
```

```bash
!(function webpackMissingModule() { var e = new Error("Cannot find module '@gitbeaker/browser'"); e.code = 'MODULE_NOT_FOUND'; throw e;
```

That's why I am reverting here, if we can able to solve this problem, I don't need gitbeaker/node, which means I don't need to update nodejs😄

Please help me, it is very necessary, otherwise I need to continue with nodeIntegration: true😥

--- a/packages/gitbeaker-browser/package.json
+++ b/packages/gitbeaker-browser/package.json
@@ -33,7 +33,7 @@
     "typescript": "^3.9.5"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=10.19.0"
   },
   "files": [
     "dist"

--- a/packages/gitbeaker-cli/package.json
+++ b/packages/gitbeaker-cli/package.json
@@ -29,7 +29,7 @@
     "typescript": "^3.9.3"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=10.19.0"
   },
   "files": [
     "dist"

--- a/packages/gitbeaker-core/package.json
+++ b/packages/gitbeaker-core/package.json
@@ -26,7 +26,7 @@
     "typescript": "^3.9.3"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=10.19.0"
   },
   "files": [
     "dist"

--- a/packages/gitbeaker-node/package.json
+++ b/packages/gitbeaker-node/package.json
@@ -25,7 +25,7 @@
     "typescript": "^3.9.3"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=10.19.0"
   },
   "files": [
     "dist"

--- a/packages/gitbeaker-requester-utils/package.json
+++ b/packages/gitbeaker-requester-utils/package.json
@@ -22,7 +22,7 @@
     "typescript": "^3.9.3"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=10.19.0"
   },
   "files": [
     "dist"

